### PR TITLE
Esign bunch of layout updates and a few other fixes

### PR DESCRIPTION
**Pull request by tkenakka**
- Deployed to test server
- Attachments on mobile not implemented, they are listed if switching to use selectionList directive but meetings have to be updated to use it + directive support for mobile and icons
- Notice block content padding added for prototyping, compare e.g. meeting proposals and signing statuses

### CODE REVIEW

File: app/directives/docsigners/docsigners.directive.js [hunk 1]
```diff
@@ -10,56 +10,88 @@
...
+        self.initiatorRoleStrId = function (docType) {
+            // Special case for request status list roles, initiator role doesn't have a role code
+            return docType && docType === CONST.ESIGNTYPE.OFFICIAL.value ? 'STR_SROLE_PREP' : CONST.ESIGNROLE.SEC.stringId;
         };
-    }]);
+
+        /* Resolve css class for signing status */
+        self.statusStyle = function (status) {
+            var s = Utils.objWithVal(CONST.ESIGNSTATUS, 'value', status);
+            return s ? s.badgeClass : 'label-default';
```
Comment (evidentti) [inline app/directives/docsigners/docsigners.directive.js]: check is s object. Utils.objWithVal can return with null.

Comment (evidentti) [inline app/directives/docsigners/docsigners.directive.js:77]: Is there a need to check is s object?

**Comment by evidentti** [comment]
LGTM

File: app/directives/docsigners/docsigners.directive.js (hunk 1 above)
Comment (tkenakka) [inline app/directives/docsigners/docsigners.directive.js:77]: Thanks, will fix and move to utils in a next PR.

**Pull request merged**